docs/source/sensors/Clementine.rst

```diff
@@ -0,0 +1,2 @@
+Clementine
+==========
```

Comment:
Clementine had 4(?) sensors. Should this heading be Clementine [sensor] to be consistent with the other headings?